PerformanceTests/DiagnosticApplication.cs

```diff
@@ -38,12 +37,11 @@ private DiagnosticApplication(TestBase testClass, Action testMethod)
 			this._testMethod = testMethod;
 			_mainWindow = new VSEmbed.DemoApp.EditorWindow();
 			_testClass.AttachToHost(_mainWindow);
-			_testClass.SetupHost();
 		}
 
 		protected override void OnStartup(StartupEventArgs e)
 		{
-			_mainWindow.Show();
+			//for (int i = 0; i < 10; i++)
```

Comment:
Some tests can be ran multiple times. I added a comment, and added another debug-time line